scripts/ci/lib.sh

```diff
@@ -516,13 +516,60 @@ poll_for_system_test_images() {
     touch "${STATE_IMAGES_AVAILABLE}"
 }
 
-image_prefetcher_start() {
+image_prefetcher_prebuilt_start() {
+    info "Starting image pre-fetcher of pre-built images (NOT built for current commit)."
+    junit_wrap image-prefetcher-prebuilt-start \
+               "Start image pre-fetcher for pre-built images (NOT built for current commit)." \
+               "See log for error details." \
+               _image_prefetcher_prebuilt_start
+}
+
+image_prefetcher_system_start() {
+    info "Starting image pre-fetcher of system images (built for current commit)."
+    junit_wrap image-prefetcher-system-start \
+               "Start image pre-fetcher for system images (built for current commit)." \
+               "See log for error details." \
+               _image_prefetcher_system_start
+}
+
+_image_prefetcher_prebuilt_start() {
+    case "$CI_JOB_NAME" in
+    gke*qa-e2e-tests)
+        image_prefetcher_start_set qa-e2e
+        ;;
+    # TODO(ROX-20508): for operaror-e2e jobs, pre-fetch images of the release from which operator upgrade test starts.
+    *)
+        info "No pre-built image prefetching is currently performed for: ${CI_JOB_NAME}"
+        ;;
+    esac
+}
+
+_image_prefetcher_system_start() {
+    case "$CI_JOB_NAME" in
+    *-operator-e2e-tests|gke*qa-e2e-tests)
```

Comment:
Actually I only noticed the GKE prefix now. Is there anything holding us from enabling it from other qa-e2e jobs as well?
```suggestion
    *-operator-e2e-tests|*qa-e2e-tests)
```

Reply:
It would also be good for more coverage.